artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/JsonUtil.java

```diff
@@ -295,6 +295,13 @@ else if (jsonValue == JsonValue.FALSE) {
             return jsonValue.toString();
          }
       }
+      else if (jsonValue instanceof Object[]) {
+         Object[] array = (Object[]) jsonValue;
+         for (int i = 0; i < array.length; i++) {
+            array[i] = convertJsonValue(array[i], desiredType);
```

Comment:
Yes, I added this because of a real failure.  The method wouldn't handle Object[] before this so would fail for management methods that returned, e.g. String[].